config/config.go

```diff
@@ -118,6 +118,8 @@ type Config struct {
 	InnkeeperPreRouteFilters  string               `yaml:"innkeeper-pre-route-filters"`
 	InnkeeperPostRouteFilters string               `yaml:"innkeeper-post-route-filters"`
 	RoutesFile                string               `yaml:"routes-file"`
+	RemoteRoutesFile          string               `yaml:"remote-routes-file"`
```

Comment:
`routes-url` maybe?

Reply:
The point is that it is not necessary the file and we do not have to follow the internal client naming convention, the command line arguments are kind of user interface

Reply:
Fine by me.